pimcore/lib/Pimcore/WorkflowManagement/Workflow/Manager.php

```diff
@@ -678,7 +703,7 @@ private function unregisterActionEvents()
 
     /**
      * Returns whether or not an element has a workflow
-     * @param Asset|ConcreteObject|Document $element
+     * @param AbstractElement $element
```

Comment:
Can't agree with you on this, since the parameter is declared as AbstractElement $element. So Doctype only allows one of these Asset|ConcreteObject|Document, but method takes AbstractElement anyway.

Your suggestion has a good point, but not actually right, don't you think so?